README.md

```diff
@@ -124,7 +124,8 @@ The container image includes a few lists proven to be successfully executed with
 
 * [openshift-conformance-minimal](tests-lists/openshift-conformance-minimal.txt). Based on the openshift/conformance/parallel/minimal tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance/parallel/minimal" | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
 * [openshift-conformance](tests-lists/openshift-conformance.txt). Based on the openshift/conformance tests suite (`openshift-tests run all --dry-run 2>/dev/null| grep "openshift/conformance | grep -v -i "clusteradmin"` and extracted the ones that passed with non cluster-admin permissions)
```

Comment:
Can you modify this as well?

Reply:
Done! We cannot modify for openshift/conformance/minimal as it is not a part of the main test suit. So modified for openshift/conformance only.